EventListener/ContentSecurityPolicyListener.php

```diff
@@ -28,8 +30,10 @@ public function onKernelResponse(FilterResponseEvent $e)
         }
 
         $response = $e->getResponse();
-        $response->headers->add($this->buildHeaders($this->report, true, $this->compatHeaders));
-        $response->headers->add($this->buildHeaders($this->enforce, false, $this->compatHeaders));
+        if (empty($this->hosts) || (!empty($this->hosts) && in_array($_SERVER['HTTP_HOST'], $this->hosts))) {
```

Comment:
The correct way (in a Symfony application) is to use request information instead of supper-globals.
`$e->getRequest()->server->get('HTTP_HOST')`